### ISSUE TITLE
Make demo of register Form, with more accessibility and user experience

### ISSUE THREAD

**Issue by shyanta** [part-up/part-up#995]
The register form lacks of accessibility. No visible tab navigation, or :focus styles. Create a flow that's easy to understand for the users. Put labels above the input fields, instead of placeholders. This to make the user experience easier.

**Comment by ralphboeije** [comment]
Hi @shyanta great that you are working on Part-up this week. As the product owner of Part-up, I am more than happy to answer any question you might have. Good to you take on this issue, don't forget UX is more than being able to use tabs.

**Comment by shyanta** [comment]
The demo's I made can be found at the following repo:
https://github.com/shyanta/part-up/tree/register-demo/register-Demo

This repo includes a readme which explains what I changed, and why I changed it. If there are any questions, let me know. I'll be happy to help you.
